config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```diff
@@ -92,4 +92,5 @@ periodics:
           --down \;
           --test=ginkgo \;
           -- \;
-          --focus-regex='\[Conformance\]'
+          --focus-regex='\[Conformance\]';
+          --use-built-binaries
```

Comment:
IMHO once this is validated this should actually be the default unless test binaries have not been built.

Reply:
+1, yeah i didn't change it in the initial PR since that would break the current default.